packages/atomic/tsconfig.json

```diff
@@ -24,13 +24,16 @@
         }
       }
     ],
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
This has an impact on the generated package output.

Would it be possible to have a separate tsconfig just for cypress tests ?

Reply:
Sure. The change was so that I could import the body of the search-api response which I'd saved in a .json file. I'll look into using a separate config. Alternatively, I could use JS instead of TS for the cypress test file.